homeassistant/components/zha/strings.json

```diff
@@ -810,6 +831,56 @@
       },
       "keypad_lockout": {
         "name": "Keypad lockout"
+      },
+      "exercise_day_of_week": {
+        "name": "Exercise day of the week",
+        "state": {
+          "sunday": "[%key:common::time::sunday%]",
+          "monday": "[%key:common::time::monday%]",
+          "tuesday": "[%key:common::time::tuesday%]",
+          "wednesday": "[%key:common::time::wednesday%]",
+          "thursday": "[%key:common::time::thursday%]",
+          "friday": "[%key:common::time::friday%]",
+          "saturday": "[%key:common::time::saturday%]"
+        }
+      },
+      "valve_orientation": {
+        "name": "Valve orientation",
+        "state": {
+          "horizontal": "Horizontal",
+          "vertical": "Vertical"
+        }
+      },
```

Comment:
These (and more) are all translated enums states, so that assumes https://github.com/home-assistant/core/pull/109309 is merged before this, right?